### PR TITLE
Hdrp decals emissive (no shader graph)

### PR DESCRIPTION
**Pull request by Paulm-Unity**
### Purpose of this PR
Add emissive property to decals

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=hdrp-decals-emissive&unity_branch=trunk&automation-tools_branch=master

**Manual Tests**: What did you do?

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?

**Halo Effect**: None, Low, Medium, High?

---
### Comments to reviewers
Shader graph implementation coming in separate PR

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassDecalForwardEmissive.hlsl [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#if (SHADERPASS != SHADERPASS_FORWARD_EMISSIVE_PROJECTOR) && (SHADERPASS != SHADERPASS_FORWARD_EMISSIVE_MESH)
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassDecalForwardEmissive.hlsl]: Please avoid to duplicate all this code here. 

you just need to #ifdef based on (SHADERPASS != SHADERPASS_FORWARD_EMISSIVE_PROJECTOR) && (SHADERPASS != SHADERPASS_FORWARD_EMISSIVE_MESH) inside ShaderPassDBuffer.hlsl, there is only few line that differ (mainly HTile and the emissive output.

Rename the file ShaderPassDBuffer.hlsl to ShaderPassDecal.hlsl instead so the name is less confusing.

Thanks

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassDecalForwardEmissive.hlsl]: Additional, please compare to the latest source as VR have added some macro in DBuffer that need to be replicate here (which will be solve if you share the code)

Comment (Paulm-Unity) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassDecalForwardEmissive.hlsl]: Done renaming and removing duplicate code, but can not find any new VR macro, which change list is it in?

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPassDecalForwardEmissive.hlsl]: I was refering to UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX, all good, it is here

**Comment by sebastienlagarde** [comment]
Katana is green, merging

**Comment by sebastienlagarde** [comment]
Next step, shader graph :)

**Comment by Paulm-Unity** [comment]
Yes sir! =)

**Pull request merged**